src/Card/Card.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+export const Card = ({ book }) => <article>
+      <h2>{book.title}</h2>
+      <p>{book.pages}</p>
```

Comment:
will consult with @ninesky00 @catherinemdean15  re what fields we'll have available here.

Eventually, we'll want to access

- book cover/image?
- book author?
- book genre?
- book age-group?
- user friends reading this book?
- comments from friends/teacher on book?